routes/api/v1/favorites.js

```diff
@@ -54,5 +58,54 @@ router.post("/", function(req, res) {
     })
 
     .catch(error => res.send({error}));
-})
+});
+
+router.get("/", function(req, res) {
+  User.findOne({
+    where: {
+      api_key: req.body.api_key
+    },
+    include: [Location]
+  })
+    .then( user => {
+      if(!user){
```

Comment:
If you wanted to cut back on a couple lines here, the ternary operator is pretty for simple if/else conditions. It would be like:
`!user ? res.sendStatus(401) :`
and then your "else" clause would start on the next line.

Reply:
Thank you! Great use of that operator, just changed it